js/script.js

```diff
@@ -0,0 +1,31 @@
+document.addEventListener('DOMContentLoaded', function() {
+    const emojiContainer = document.getElementById('emojiContainer');
+    const emojis = ['😻', '🍔', '🔥', '🐣', '💩'];
+
+    const emojiCounts = {};
+
+    emojis.forEach(emoji => {
+        const emojiBox = document.createElement('div');
+        emojiBox.classList.add('emoji-box');
+        
+        const emojiElement = document.createElement('span');
+        emojiElement.classList.add('emoji');
+        emojiElement.textContent = emoji;
+
+        const countElement = document.createElement('span');
+        countElement.classList.add('emoji-count');
+        countElement.textContent = '0';
+
+        emojiBox.appendChild(emojiElement);
+        emojiBox.appendChild(countElement);
+
+        emojiContainer.appendChild(emojiBox);
+
+        emojiCounts[emoji] = countElement;
+
+        emojiElement.addEventListener('click', function() {
```

Comment:
Adding a listener for each emoji, but should be one listener for all

Look at this solution

```
<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Voting smiles</title>
    <style>
        .parent {
            display:flex;
            margin: 30px 0 0;
            justify-content: center;
            user-select: none;
        }

        .emoji {
            padding: 10px;
            font-size: 60px;
        }

        .emoji__picture {
            cursor: pointer;
        }

        .emoji__count {
            text-align: center;
        }
    </style>
</head>
<body>
    <div class="parent">
        <div class="emoji">
            <div class="emoji__picture">&#129315;</div>
            <div class="emoji__count">0</div>
        </div>
        <div class="emoji">
            <div class="emoji__picture">&#128578;</div>
            <div class="emoji__count">0</div>
        </div>
        <div class="emoji">
            <div class="emoji__picture">&#128528;</div>
            <div class="emoji__count">0</div>
        </div>
        <div class="emoji">
            <div class="emoji__picture">&#128577;</div>
            <div class="emoji__count">0</div>
        </div>
        <div class="emoji">
            <div class="emoji__picture">&#128557;</div>
            <div class="emoji__count">0</div>
        </div>
        <!-- UNCOMMENT THIS CODE -->
        <!--<div class="emoji">
            <div class="emoji__picture">&#129488;</div>
            <div class="emoji__count">0</div>
        </div>-->
    </div>
    <script>
      const parent = document.querySelector(".parent");

      parent.addEventListener("click", (event) => {
        if (event.target.closest(".emoji__picture")) {
          const nextElementSibling = event.target.nextElementSibling;

          nextElementSibling.textContent = Number(nextElementSibling.textContent) + 1;
        }
      });
    </script>
</body>
</html>
```

Reply:
One more option
```
          const nextElementSibling = event.target.parentElement.querySelector(".emoji__count");

          nextElementSibling.textContent = Number(nextElementSibling.textContent) + 1;
```